app/code/Tasks/FeedbackManager/Block/Adminhtml/FeedbackView.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Tasks\FeedbackManager\Block\Adminhtml;
+
+class FeedbackView extends \Magento\Framework\View\Element\Template
+{
+    protected $feedbackCollection;
+
+    protected $request;
+
+    // protected $button;
+
+    public function __construct(
+        \Magento\Framework\View\Element\Template\Context $context,
+        \Magento\Framework\App\Request\Http $request,
+        // \Magento\Backend\Block\Widget\Form\Container $button,
+        \Tasks\FeedbackManager\Model\ResourceModel\Feedback\CollectionFactory $feedbackCollection,
+        array $data = []
+    ) {
+        $this->feedbackCollection = $feedbackCollection;
+        $this->request = $request;
+        // $this->button = $button;
```

Comment:
@gayathri-aravind Please remove unwanted and commented out codes from PR

Reply:
Removed the unwanted and commented codes from PR. Commit : 3510754a80bfe52742e2d2b8bc43fd4bf7e4ed8e